.gitlab-ci.yml

```diff
@@ -326,9 +326,20 @@ benchmarks-statemint:
     - ./scripts/benchmarks-ci.sh assets westmint ./artifacts
     - export BRANCHNAME="weights-statemint-${CI_COMMIT_BRANCH}"
     - *git-commit-push
+    # create PR to ${CI_COMMIT_BRANCH}
+    - curl -u ${GITHUB_USER}:${GITHUB_TOKEN}
+      -d '{"title":"[benchmarks] Update weights for statemine/t","body":"This PR is generated automatically by CI.","head":"'${BRANCHNAME}'","base":"'${CI_COMMIT_BRANCH}'"}'
+      -X POST https://api.github.com/repos/paritytech/${CI_PROJECT_NAME}/pulls
+    # create PR to master
+    - curl -u ${GITHUB_USER}:${GITHUB_TOKEN}
+      -d '{"title":"[benchmarks] Update weights for statemine/t","body":"This PR is generated automatically by CI.","head":"'${BRANCHNAME}'","base":"master"}'
+      -X POST https://api.github.com/repos/paritytech/${CI_PROJECT_NAME}/pulls
+    # create PR to a branch with version number (e.g. v0.9.270)
+    # transform release-parachains-v9270 to v0.9.270
+    - export BASEBRANCH=$(echo ${CI_COMMIT_BRANCH} | cut -d "-" -f 3 | sed -e "s/\(.\)\(.\)\(...\)/\10.\2.\3/")
```

Comment:
As long as we have native runtimes, we should add a 4th and have the PR against both the branches:
- `release-parachains-v9270` (runtimes branch)
- `release-v0.9.270` (node branch)